pkg/kubectl/cmd/taint.go

```diff
@@ -366,24 +379,21 @@ func validateNoTaintOverwrites(obj runtime.Object, taints []v1.Taint) error {
 	return utilerrors.NewAggregate(allErrs)
 }
 
-// updateTaints updates taints of obj
-func (o TaintOptions) updateTaints(obj runtime.Object) error {
+// updateTaints applies a taint option(o) to a node in cluster after validating conditions like overwriting of option and returns a human understandable description of the said option result.
```

Comment:
`+// updateTaints applies a taint option (o) to a node in the cluster, and after computing the net effect of the operation (i.e. does it result in an overwrite?), it reports the end result back in a way that a user can easily interpret.`